src/test/ui/resolve/enums-are-namespaced-xc.stderr

```diff
@@ -14,7 +16,9 @@ error[E0425]: cannot find function `B` in module `namespaced_enums`
    |                               ^ not found in `namespaced_enums`
    |
 help: possible candidate is found in another module, you can import it into scope
-   | use namespaced_enums::Foo::B;
+   |
+12 | use namespaced_enums::Foo::B;
+   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
if the entire span is the suggestion, I don't think it should put the `^^^` under it